### ISSUE TITLE
[WPF] FunctionSeries.Title does not show

### ISSUE THREAD

**Issue by esoemad5** [oxyplot/oxyplot#2114]
### Steps to reproduce

1. Copy the code from https://oxyplot.readthedocs.io/en/latest/getting-started/hello-wpf.html
2. Run the app

Platform:
.NET version: .NET 8.0
I noticed the [HelloWorld example](https://github.com/oxyplot/documentation-examples/blob/master/HelloWorld/WpfApplication1/WpfApplication1.csproj) was .NET Core 3.0  I tried that version and found the same issue

### Expected behavior

A plot of a cosine curve shows with a legend identifying the curve as "cos(x)"

### Actual behavior

A plot of a cosine curve shows, but no legend is present

<img width="654" alt="Screen Shot 2024-11-22 at 9 35 07 PM" src="https://github.com/user-attachments/assets/6013b27c-971f-4a87-a296-a32d4d1e1245">
<img width="713" alt="Screen Shot 2024-11-22 at 9 35 39 PM" src="https://github.com/user-attachments/assets/b60a9d5f-0e46-4ba5-b691-ebfa07cdd8d6">
<img width="883" alt="Screen Shot 2024-11-22 at 9 35 54 PM" src="https://github.com/user-attachments/assets/cdfe3cd3-4bf1-4eab-96a4-e2985e154516">

**Comment by VisualMelon** [comment]
Yeah, the docs are out of date: you need to add a `Legend` to the plot. See https://github.com/oxyplot/oxyplot/blob/develop/Source/Examples/ExampleLibrary/Examples/LegendExamples.cs for examples, but `this.MyModel.Legends.Add(new Legend())` should get something simple showing.
